package.json

```diff
@@ -52,7 +54,8 @@
     "metro-react-native-babel-preset": "^0.59.0",
     "prettier": "^2.0.2",
     "react": "16.9.0",
-    "react-native": "0.61.5",
+    "react-native": "0.62",
```

Comment:
This is likely going to break the Android and iOS examples if we don't upgrade them. Please verify they still work. We should also ask for 0.62.3 here.

Reply:
there are CircleCI jobs in this repo that validate these build so I'm going off of those. I'm not able to validate this on mac or android, it sounds like you are, if so please help validate it.

Reply:
There also doesn't seem to be a 0.62.3:
![image](https://user-images.githubusercontent.com/22989529/92228315-8ab8f300-ee5c-11ea-85ba-68d307e79766.png)

Reply:
Sorry meant 0.62.2

Reply:
But lets get you access to a Mac then. I'll start a thread 🙂